packages/cli-dashboard/public/index-production.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
This means the other `index.html` is used for development only? Let's keep the production file name as `index.html` and the development file's name may be something like `index-dev.html`

Reply:
Alternatively, there can be only one `index.html` file and `<script src="index.js"></script>` can be injected conditionally.

Reply:
Done by using `index.ejs`